branding/celo/src/icons/WelcomeLogo.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
```

Comment:
Wasn't sure what the placeholder value should be so just copied over from Logo.tsx and updated the props to match the real component